cmd/docker/docker.go

```diff
@@ -54,7 +54,7 @@ func newDockerCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags = cmd.Flags()
 	flags.BoolVarP(&opts.Version, "version", "v", false, "Print version information and quit")
 	flags.StringVar(&opts.ConfigDir, "config", cliconfig.Dir(), "Location of client config files")
-	opts.Common.InstallFlags(flags)
+	opts.Common.InstallFlags(flags, cmd.PersistentFlags())
```

Comment:
I don't get this change, this feels really weird to add the flag twice (on `cmd.Flags()` and `cmd.PersistentFlags()`)

Reply:
"orchestrator" is the only flag added to cmd.PersistentFlags() in InstallFlags, all the other flags are still in cmd.Flags()

Reply:
Slightly confused as well

Reply:
OK, I'll just remove the related commit.